src/src/Static/WebStaticData.cpp

```diff
@@ -150,4 +150,52 @@ void serve_JS(JSfiles_e JSfile) {
     html_add_script(false);
     streamFromFS(fname);
     html_add_script_end();
+}
+void serve_CMJS(CMfiles_e CMfile) {
```

Comment:
Why a new function which does essentially the same as the one for the other JS files?

Reply:
because i am still learning and this is my amateurish attempt to integrate the files. And i thought i might be good to to have a separation to everything else.....🤷🏻‍♂️

Reply:
It has 2 main drawbacks:

- Code duplication
- Increase build size

The latter one is specific an issue when working with microcontrollers.
But the first one is (IMHO) always a concern as you may also duplicate bugs or introduce bugs if you need to change something but don't change it for all occurrences.

But I think it is better to merge it and then let me or @tonhuisman deduplicate it.

Reply:
> But I think it is better to merge it and then let me or @tonhuisman deduplicate it.

Thats a good plan. 🙂